modules/aws/sonar-base-instance/outputs.tf

```diff
@@ -23,6 +23,24 @@ output "sg_id" {
   value       = aws_security_group.dsf_base_sg.id
 }
 
+output "ingress_ports" {
+  value = local.ingress_ports
+  description = "The ingress ports of the security group on the DSF node EC2"
+}
+
+output "iam_role" {
+  description = "IAM Role ARN of the DSF node"
+  value = local.role_arn
+}
+
+output "primary_node_sonarw_public_key" {
```

Comment:
Same issue with primary I mentioned in another comment

Reply:
Renamed to sonarw_public_key and sonarw_private_key